pkgs/applications/window-managers/hyprwm/hyprpaper/default.nix

```diff
@@ -2,28 +2,39 @@
 , stdenv
 , fetchFromGitHub
 , cmake
+, cairo
+, expat
 , file
+, fribidi
 , hyprlang
+, libdatrie
 , libGL
 , libjpeg
+, libselinux
+, libsepol
+, libthai
 , libwebp
-, mesa
+, libXdmcp
 , pango
+, pcre
+, pcre2
 , pkg-config
+, util-linux
 , wayland
 , wayland-protocols
 , wayland-scanner
+, debug ? false
```

Comment:
why make this argument? im sure those who want to specifically build a debug version can use `hyprpaper.overrideAttrs { cmakeBuildType = "Debug"; }` over `hyprpaper.override { debug = true; }`

Reply:
Makes sense, I'll remove it then.